apps/src/blockly/components/ModalFunctionEditor.jsx

```diff
@@ -18,12 +35,16 @@ export default function ModalFunctionEditor() {
     >
       <div className={classNames('toolbar', moduleStyles.toolbar)}>
         <div className={moduleStyles.buttons}>
-          <button type="button" id={MODAL_EDITOR_DELETE_ID}>
-            delete
-          </button>
-          <button type="button" id={MODAL_EDITOR_CLOSE_ID}>
-            close
-          </button>
+          {renderButton(
```

Comment:
This is an interesting pattern. `renderButton` feels pretty non-specific as a helper function to be nested inside of `ModalFunctionEditor`... Any thoughts on how we could avoid that? Should we move it to a util? I'm also wondering if it saves enough repetition to be worth just rendering two `Button`s.

Reply:
My goal was to not have to repeat the onClick and button size, but that may not be worth a helper function (I think at one point I had more repeat code). Is the concern that it's not enough of a helper or that I should rename? I could rename to `renderNarrowButton` or something.

Reply:
I assumed it did more than it did because it had a dedicated helper, and I wonder if it may not be worth a helper function. I think repeating the onClick and button size is fine, or maybe creating a `const editorButtonSize = Button.ButtonSize.narrow` and then repeating it? (Not that that's shorter or even necessary, but maybe it connects the two?)

Reply:
got rid of the helper!